tests/unit/test_adapter.py

```diff
@@ -638,7 +645,10 @@ def test_parse_relation_with_statistics(self):
             ("Location", "/mnt/vo"),
             ("Serde Library", "org.apache.hadoop.hive.serde2.lazy.LazySimpleSerDe"),
             ("InputFormat", "org.apache.hadoop.mapred.SequenceFileInputFormat"),
-            ("OutputFormat", "org.apache.hadoop.hive.ql.io.HiveSequenceFileOutputFormat"),
+            (
+                "OutputFormat",
+                "org.apache.hadoop.hive.ql.io.HiveSequenceFileOutputFormat",
+            ),
```

Comment:
I would love to figure out what's different in my VS Code settings from you guys...I don't mean to reformat random files in every PR :)